test/simple-neurode.jl

```diff
@@ -0,0 +1,48 @@
+using Flux, DiffEqFlux, OrdinaryDiffEq
+using Zygote
+
+u0 = Float32[2.; 0.]
+datasize = 30
+tspan = (0.0f0,1.5f0)
+
+function trueODEfunc(du,u,p,t)
+    true_A = [-0.1 2.0; -2.0 -0.1]
+    du .= ((u.^3)'true_A)'
+end
+t = range(tspan[1],tspan[2],length=datasize)
+prob = ODEProblem(trueODEfunc,u0,tspan)
+ode_data = Array(solve(prob,Tsit5(),saveat=t))
+
+dudt = Chain(x -> x.^3,
+             Dense(2,50,tanh),
+             Dense(50,2))
+#= ps = Flux.params(dudt) =#
+
+p = DiffEqFlux.destructure(dudt)
+ps = Flux.params(p)
+dudt_(u,p,t) = DiffEqFlux.restructure(dudt,p)(u)
+prob = ODEProblem(dudt_,u0,tspan,p)
+
+diffeq_adjoint(p,u0,prob,Tsit5())
+
+forward(u->diffeq_adjoint(p,u,prob,Tsit5()),u0)
```

Comment:
@MikeInnes evaluates, but evaluating the pullback gives the variable not defined error.

```
f_, back = forward(u->diffeq_adjoint(p,u,prob,Tsit5()),u0)
back(f_) #ERROR: Variable %46 in block 8 is not defined.
```

Also this is true for `p` as well:
`f_, back = forward(p_->diffeq_adjoint(p_,u0,prob,Tsit5()),p)`

Reply:
@MikeInnes correction, with updates to Zygote#master this now gives `ERROR: Can't differentiate foreigncall expression`